bin/stafftools

```diff
@@ -37,6 +37,36 @@ class JiraStaffTools < Thor
     output_json(response.body)
   end
 
+  desc "current <git_hub_installation_id>", "Display the current sync item"
+  def current(git_hub_installation_id)
+    jira_host = get_jira_host(git_hub_installation_id)
+    response = client.get(
+      "/api/#{CGI.escape(git_hub_installation_id)}/repoSyncState.json",
+      "jiraHost" => jira_host
+    )
+
+    foundIt = false
+    repos = JSON.parse(response.body)["repos"]
+    doneCount = 0
+
+    repos.each do |repo_id, r|
+      if [r["pullStatus"], r["branchStatus"], r["commitStatus"]] == (["complete"] * 3)
```

Comment:
What is this doing?

Reply:
if all three keys have a value of "complete", we know that this repo has been fully synced, if not, it's the one we want because only one repo is "in progress" at a time.

Reply:
OH its completed 3 times in an array. Id recommend `%w(completed completed completed)` to be more obvious